lib/http-bridge/test/integration/Main.hs

```diff
@@ -71,13 +77,17 @@ main = hspec $ do
     describe "Cardano.WalletSpec" Wallet.spec
     describe "Cardano.Wallet.HttpBridge.NetworkSpec" HttpBridge.spec
     describe "CLI commands not requiring bridge" MnemonicsCLI.spec
-    beforeAll startCluster $ afterAll killCluster $ after tearDown $ do
+    beforeAll (startCluster ListenOnRandomPort) $ afterAll killCluster $ after tearDown $ do
         describe "Wallets API endpoint tests" Wallets.spec
         describe "Transactions API endpoint tests" Transactions.spec
         describe "Addresses API endpoint tests" Addresses.spec
         describe "Wallets CLI tests" WalletsCLI.spec
         describe "Transactions CLI tests" TransactionsCLI.spec
         describe "Addresses CLI tests" AddressesCLI.spec
+
+    let defaultPort = 8090
+    beforeAll (startCluster (ListenOnPort defaultPort)) $ afterAll killCluster $ after tearDown $ do
```

Comment:
Let's try to keep lines to the length specified by https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters